public-images/public-images-how-to/run-an-ova-using-virtualbox.rst

```diff
@@ -0,0 +1,258 @@
+setup of virtual machines (VMs). VirtualBox is an open source
+virtualisation platform that allows you to use OVA files to create and
+manage VMs. This guide will cover how to acquire an OVA file,
+install VirtualBox, and configure and launch an Ubuntu VM that you can connect
+to via the virtual console or over SSH.
+
+Getting an OVA
+--------------
+
+The easiest way to get an OVA is downloading one from
+`cloud-images.ubuntu.com <http://cloud-images.ubuntu.com>`__. You can
+get the latest daily OVA for a given
+``$SERIES`` by downloading ``https://cloud-images.ubuntu.com/$SERIES/current/$SERIES-server-cloudimg-amd64.ova``.
+
+We will create a new directory and get the latest daily for Ubuntu 24.04
+LTS (Noble Numbat).
+
+.. code:: bash
+
+  mkdir virtualbox-ova
+  cd virtualbox-ova/
+  wget https://cloud-images.ubuntu.com/noble/current/noble-server-cloudimg-amd64.ova
+
+
+Getting VirtualBox
+------------------
+
+There are two ways of getting VirtualBox depending on whether or not you
+are on an Ubuntu system.
+
+From the Ubuntu archive
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are on Ubuntu, the simplest way to get VirtualBox is through the
+Ubuntu archive, the official repository provided by Ubuntu.
+
+.. code:: bash
+
+  sudo apt update
+  sudo apt install virtualbox
+
+From the upstream source
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Alternatively, you can download VirtualBox from the `upstream
+source <https://www.virtualbox.org/wiki/Downloads>`__. This page
+provides instructions for installing VirtualBox on your OS, so they will
+not be covered here.
+
+Importing the OVA in VirtualBox
+-------------------------------
+
+Once the installation is complete you can start VirtualBox.
+
+.. image:: run-an-ova-using-virtualbox-images/0-launch-virtualbox.png
+   :align: center
+
+Click **Import** or select **File > Import Appliance...** from the menu bar to add our OVA as a virtual appliance.
+
+.. image:: run-an-ova-using-virtualbox-images/1-import-appliance.png
+   :align: center
+
+Click **Next** to proceed once you have selected your file.
+
+.. image:: run-an-ova-using-virtualbox-images/2-appliance-settings.png
+   :align: center
+
+From here, we can modify the settings of the appliance. Double-click
+on the items to change properties, or enable/disable applicable
+properties using the check boxes. We will leave everything as per the
+suggested settings. Click **Import** to continue.
+
+.. image:: run-an-ova-using-virtualbox-images/3-imported-vm.png
+   :align: center
+
+Now from the VirtualBox home screen, we can select the imported
+appliance from the left sidebar. Before we attempt to run the appliance however,
+we will need to generate and attach an ISO image containing
+configuration data for cloud-init to initialise the instance during
+initial boot.
+
+Adding the local data store
+---------------------------
+
+To create the ISO image we will use the ``cloud-localds`` function from
```

Comment:
I just added an article on creating the local ds, you could link to that if you wanted.

Reply:
Thanks for this. I've added a link above for users to get more information. Didn't replace the whole section so that users wouldn't need to click away to complete the step, and to avoid unnecessary installation of `qemu-system-x86` if they copy-paste the terminal commands in the article.